pkg/config/autodiscovery/autodiscovery.go

```diff
@@ -55,6 +55,12 @@ func DiscoverComponentsFromEnv() ([]config.ConfigurationProviders, []config.List
 		log.Info("Adding Container provider from environment")
 	}
 
+	if config.IsFeaturePresent(config.ECSFargate) {
+		detectedProviders = append(detectedProviders, config.ConfigurationProviders{Name: names.Container, Polling: true, PollInterval: "1s"})
+		detectedListeners = append(detectedListeners, config.Listeners{Name: names.ECS})
```

Comment:
The "names" package is only for providers (`pkg/autodiscovery/providers/names`). We should probably have a dedicated one for the listeners.

Not for now, but let's keep this in mind for a future refactor PR.